Signal/src/view controllers/FullImageViewController.m

```diff
@@ -147,6 +147,10 @@ - (void)initializeImageView {
         self.imageView.userInteractionEnabled       = YES;
         self.imageView.clipsToBounds                = YES;
         self.imageView.layer.allowsEdgeAntialiasing = YES;
+        // Use trilinear filters for better scaling quality at
+        // some performance cost.
+        self.imageView.layer.minificationFilter = kCAFilterTrilinear;
+        self.imageView.layer.magnificationFilter = kCAFilterTrilinear;
```

Comment:
We definitely want to use the highest quality scaling in the image attachment view.